docs/reference/analysis/common-uses.asciidoc

```diff
@@ -0,0 +1,98 @@
+== Common uses for analysis
+++++
+<titleabbrev>Common uses</titleabbrev>
+++++
+
+Analyzing full text often means analyzing natural, human language. While each
+language is different, this usually means solving a common set of problems for
+search, such as how to reduce words to their root form.
+
+This section describes how {es} is commonly used to address these problems and
+our recommendations.
+
+[TIP]
+====
+If you'd like to get started quickly, we recommend using {es}'s built-in
+<<analysis-standard-analyzer,`standard`>> analyzer or 
+<<analysis-lang-analyzer,language analyzers>>. These analyzers provide good
+out-of-the-box support for most use cases.
+====
+
+[float]
+[[stemming]]
+=== Stemming
+
+Most natural languages are inflected, meaning words can change
+their form to express differences, such as the following:
+
+* _Number_:      fox, foxes
+* _Tense_:       pay, paid, paying
+* _Person_:      hear, hears
+
+_Stemming_ attempts to remove theses differences and reduce each token to its
+root form. For example, `foxes` can be stemmed to the root `fox`. This often
+aids search retrieval by consolidating near-identical terms.
+
+[float]
+[[stemming-token-filters]]
+==== Stemming token filters
+
+In {es}, stemming is handled by a <<analysis-tokenfilters,token filter>> in
+the <<analyzer-anatomy,analyzer>>.
+
+If you're unsure of which token filter to use for stemming, we recommend
+starting out with the <<analysis-stemmer-tokenfilter,`stemmer`>> filter, which
+supports multiple languages and stemmer types. The
+<<analysis-stemmer-tokenfilter,`stemmer` filter documentation>> page highlights
+the recommended stemmer for each language in bold, based on a reasonable
+compromise between performance and quality.
+
+If the recommended stemmer does not fit your use case, spend some time
+investigating and comparing the other available stemmers to
+find the one that best suits your needs.
+
```

Comment:
This doesn't really add anything. If basically says, "If it doesn't fit your use case, you'll need to figure it out." Some sort of guidance would be helpful. Maybe some examples of when you might opt to use one of the other stemmers?

Reply:
I've opted to just remove this with fa157a3080a for now. I feel that we could add more in-depth advice for choosing a stemmer, but  that would be a considerable work effort worthy of its own PR.